### PR TITLE
refactor(bun): lowercase -D paremeter if the agent is bun

### PR DESCRIPTION
**Pull request by eggsy**
because bun doesn't accept `-D` but does accept `-d`, stupid.
- also fixed readme issue and added a basic `.prettierrc` file
- added missing bun examples in readme.md, and put bun notices

### CODE REVIEW

**Comment by antfu** [comment]
1. I don't use Prettier, we should not have a pretterrc
2. For tests of bun, I think it's better to keep the tests but expect them to fail, so in the future, when bun supports them we could then do the update.

File: src/parse.ts [hunk 1]
```diff
@@ -29,6 +29,9 @@ export const parseNi = <Runner>((agent, args, ctx) => {
     process.exit(0)
   }
 
+  if (args.includes('-D') && agent === 'bun')
+    return getCommand(agent, 'add', args.join(' ').replace('-D', '-d').split(' '))
```
Comment (antfu) [inline src/parse.ts:33]:
```suggestion
  if (agent === 'bun')
    args = args.map(i => i === '-D' ? '-d' : i)
```

**Comment by eggsy** [comment]
didn't quite understand what you meant by "expect to fail" but added the file back, it's still throwing "command x is not supported by the agent" but it builds fine

**Comment by eggsy** [comment]
oh, got it now 🤣 can tell i never do testing

**Pull request merged**